pkg/config/config.go

```diff
@@ -762,6 +762,7 @@ func InitConfig(config Config) {
 	// Datadog security agent (common)
 	config.BindEnvAndSetDefault("security_agent.cmd_port", 5010)
 	config.BindEnvAndSetDefault("security_agent.expvar_port", 5011)
+	config.BindEnvAndSetDefault("security_agent.log_file", "")
```

Comment:
This way of defining `security_agent.log_file` is consistent with the way the `log_file` parameter is defined.

But I’m wondering why we couldn’t set the default value here.
```suggestion
	config.BindEnvAndSetDefault("security_agent.log_file", common.DefaultSecurityAgentLogFile)
```

so that we don’t need to handle the default value case each time we want to use that parameter:
```diff
  logFile := coreconfig.Datadog.GetString("security_agent.log_file")
- if logFile == "" {
-   logFile = common.DefaultSecurityAgentLogFile
- }
```
in `cmd/security-agent/api/agent/agent.go`, `cmd/security-agent/app/app.go` and `cmd/security-agent/app/flare.go`

Reply:
indeed make sens